scripts/cibuildpkg.py

```diff
@@ -115,6 +115,7 @@ class Package:
     requires: List[str] = field(default_factory=list)
     source_dir: str = ""
     source_strip_components: int = 1
+    source_code_file_name: str = ""
```

Comment:
I did wonder how long we would be able to keep going without renaming files :)

However could we have this in a separate subsequent PR, so we can hunt down all packages whose tarball should be renamed (at first glance openjpeg, libvpx)?

Reply:
Allright, see https://github.com/PyAV-Org/pyav-ffmpeg/pull/65
But also please help me on this PR ,
There are two problems.
1. some configure can not run by `dash`, it is better to use itself after checking the running permission of the configure instead of run `sh configure`.
2. the FFmpeg compile with an error ` ERROR: openh264 not found using pkg-config`,but in the `/tmp/vendor/lib/pkgconfig`, openh264.pc exists. And the build log in ffbuild/config.log did not exist.

Reply:
Point 1/ feels out of scope, the build script is there to produce wheels on a CI environment - which works fine. If you encounter problems on your local setup feel free to locally remove the "sh" part or replace it with "bash", but  it was necessary to get the build working on Windows.

For point 2/ I would really appreciate it if you investigate yourself. Every new package we integrate is tedious work, and I would expect a PR to help with the maintenance burden, not increase it.